### ISSUE TITLE
collectd fails to build in parallel. In addition, cpu not working in 5.5.0.

### ISSUE THREAD

**Issue by cactus** [Homebrew/homebrew#42343]
Two issues:
* collectd often fails to build in parallel.
* cpu metric collection not working in 5.5.0 on osx. The error is `cpu plugin: processor_info returned only 4 elements`.

This patch fixes both.

references: 
* https://github.com/collectd/collectd/issues/1146 (parallel build issue)
* http://mailman.verplant.org/pipermail/collectd/2015-May/006546.html (cpu stats issue)

```
diff --git a/Library/Formula/collectd.rb b/Library/Formula/collectd.rb
index da24134..dfb4db4 100644
--- a/Library/Formula/collectd.rb
+++ b/Library/Formula/collectd.rb
@@ -19,6 +19,13 @@ class Collectd < Formula
     depends_on "autoconf" => :build
   end
 
+  stable do
+      patch do
+        url "https://github.com/collectd/collectd/commit/e0683047a42e217c352c2419532b8e029f9f3f0a.diff"
+        sha256 "7053170a072d27465b69eed269d32190ec810bcb0db59f139a1682e71a326fdd"
+      end
+  end
+
   # Will fail against Java 1.7
   option "with-java", "Enable Java 1.6 support"
   option "with-debug", "Enable debug support"
@@ -39,6 +46,10 @@ class Collectd < Formula
   end
 
   def install
+    # collectd breaks with makejobs
+    # see: https://github.com/collectd/collectd/issues/1146
+    ENV.deparallelize
+
     args = %W[
       --disable-debug
       --disable-dependency-tracking
```

**Comment by DomT4** [comment]
Could you try and open a PR for these changes? Thanks!

**Comment by cactus** [comment]
@DomT4 done. pull req opened.

**Comment by cactus** [comment]
fixed by #42344
